common4j/src/main/com/microsoft/identity/common/java/telemetry/Telemetry.java

```diff
@@ -280,6 +280,38 @@ public void flush(@NonNull final String correlationId) {
         }
     }
 
+    /**
+     * Get the final Telemetry Map for a correlationId.
+     * @param correlationId The correlation id should either passed by the client app through the API call
+     *                      or generated by the API dispatcher.
+     */
+    public List<Map<String, String>> getFinalMap(@NonNull final String correlationId) {
```

Comment:
This method can be called at any time i.e. it returns a telemtetry map but NOT the "final" map...let's remove the word `final` from the method name

Reply:
It seems like this needs a unit test.

Reply:
Added unit test.